### DISCUSSION TITLE
Where can i view version 3.x components document?

### DISCUSSION THREAD

**Discussion by zeroven0205** [chakra-ui/chakra-ui#9220] [Q&A] [answered]
Where can i find version 3.x components document?
version 3.x is big diff from version 2.x.
only https://v2.chakra-ui.com/ doc exist,  no v3 doc yet!

**Comment by segunadebayo** [comment, marked as answer]
You can find the v3 component documentation here:
https://www.chakra-ui.com/docs/components/concepts/overview
